ee/fe/modules/500-openvpn/templates/openvpn/authenticator.yaml

```
@@ -0,0 +1,19 @@
+{{- if .Values.openvpn.internal.deployDexAuthenticator }}
+---
+apiVersion: deckhouse.io/v1
+kind: DexAuthenticator
+metadata:
+  name: "openvpn"
```

```suggestion
  name: openvpn
```
Why quotation? There aren't quotations in other templates.